### PR TITLE
new post: migrate posts from PyCon TW blogger

### PR DESCRIPTION
**Pull request by Lee-W**
## Types of changes

* new post

## Description

With minor modifications from [this awesome article](https://daniel.feldroy.com/posts/2022-02-blogger-to-markdown-script), I'm able to migrate posts from [PyCon TW blogger](https://pycontw.blogspot.com/) to here. Might need some tweak before merging

### CODE REVIEW

File: content/posts/2018-05-30-pycontw-2018-prior-notification.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+
+Registration
+============
+
+Registration time begins at 08:10 on June 1. After arriving the conference venue, please connect to the conference WiFi in order to successfully register through pressing the registration button in “PyCon TW Pass” app.
+
+Please download PyCon TW Pass via link at <https://ccip.pycon.tw/>, and open it:
+
+Please register at the reception on the 3F and follow the instrcution to collect the goodie bag.
```
Comment (mattwang44) [inline content/posts/2018-05-30-pycontw-2018-prior-notification.md:160]:
```suggestion
Please register at the reception on the 3F and follow the instruction to collect the goodie bag.
```

File: content/posts/2017-03-09-carol-willing-bio-carol-willing-is.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+
+#####
+
+### Introduction(介紹)
+
+
+#### **經常總會聽到 Carol Naslund Willing 說：「這真是太酷了。它是如何運作的？好希望學習到更多有關的事物。」 她熱愛開發軟體，目前為 PSF (Python Software Foundation) 基金會理事之一 、 Project Jupyter 的核心開發人員與加州理工大學聖路易斯奧比斯 ( Cal Poly San Luis Obispo) 的軟體研究工程師。她活躍於開源專案的共同開發，並且維護像是 OpenHatch 與 Anita Brog Institute 的開源專案。
+
+與人之間的經驗使我有了這樣感觸：「用你所學，使事情更好，並且與人分享。」Caril Wiilling 這樣說的。
```
Comment (mattwang44) [inline content/posts/2017-03-09-carol-willing-bio-carol-willing-is.md:46]:
```suggestion
與人之間的經驗使我有了這樣感觸：「用你所學，使事情更好，並且與人分享。」Caril Willing 這樣說的。
```

File: content/posts/2016-05-28-kaohsinugpy-victor.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+Title: Kaohsinug.py 聊聊－社群共同創辦人 Victor 專訪
```
Comment (mattwang44) [inline content/posts/2016-05-28-kaohsinugpy-victor.md:1]:
```suggestion
Title: Kaohsiung.py 聊聊－社群共同創辦人 Victor 專訪
```

File: content/posts/2015-06-04-10-things-you-dont-want-to-miss-at_4.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+
+For the main theme “ Back to \_\_future\_\_ II," there is a History wall at 3F, you can look on it to know the past of python and PyCon Taiwan. And, you can add your memory on it, with us.
+
+
+### Show Time & Job Fair
+
+Talents, Engineers, Sponsors, Hunters, are gathered together.
+
+Like a carnival, Show Ttime & Job Fair is a party we encourage our sponsors to show their products, culture, and even what they can do for you! All you need to do is to walk around the booth area, take a look on what's going on there. We bet you will find something really impressing.
```
Comment (mattwang44) [inline content/posts/2015-06-04-10-things-you-dont-want-to-miss-at_4.md:84]:
```suggestion
Like a carnival, Show Time & Job Fair is a party we encourage our sponsors to show their products, culture, and even what they can do for you! All you need to do is to walk around the booth area, take a look on what's going on there. We bet you will find something really impressing.
```

File: content/posts/2015-06-04-10-things-you-dont-want-to-miss-at_4.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+
+See you on Saturday at R4!
+
+
+### Night Market
+
+![](https://lh6.googleusercontent.com/qtSndggbTpwn_AaYj4xbJlosuhqbU91xzHmLJ1qKtFs0TCyBK_qZNiRvB7d0KenLjfyGxrJwdzu15o6_wAhfXKDBzHxF2LfBqWgVWz3zbaWGyXd4SDWefOq7N6YCuvJvFa6H154)![](https://lh4.googleusercontent.com/ddoKu9OckXYbWtt89NDLoRYiiR9TED_tLW9W2wYKR4MED6qOkoGPcqKIn88Gkn2K_JZ9qImHyxEc3ghrEGUqNJUZQ4hImi9QU1QxeYvjUA08EssAbj6Yw3j92D-U_znNvPpL_dE)
+
+Everyone loves Night-Market. Now you can find it on PyCon APAC 2015, within Pythoniest Style ! You can enjoy the buffet, the performance, and the boothes with genious stuffs. All made by Python.
```
Comment (mattwang44) [inline content/posts/2015-06-04-10-things-you-dont-want-to-miss-at_4.md:64]:
```suggestion
Everyone loves Night-Market. Now you can find it on PyCon APAC 2015, within Pythoniest Style ! You can enjoy the buffet, the performance, and the booths with genius staffs. All made by Python.
```

**Review by mattwang44 — COMMENTED** [review]
Some typos found by [cspell](https://www.npmjs.com/package/cspell)

**Review by 9ukei — COMMENTED** [review]
@Lee-W  I think you've done a great job, and there probably might not be any need for modifications. Thanks!

**Review by 9ukei — APPROVED** [review]
@winniepopu Please check if there's anything that needs to be modified when you have time, Thanks!

**Review by winniepopu — APPROVED** [review]
Hi Wei, I found there're some redundant spaces and lines in posts.
I used IDE to delete white space automatically and keep markdown clear format.
I've pushed to your branch, please help to check. Thanks!

**Pull request merged**